.gitignore

```diff
@@ -53,6 +53,8 @@ _site/
 .env.local
 env/
 env[23]/
+.venv
```

Comment:
I'm not sure if that is the same as `venv/`

Reply:
My .venv folder is getting ignored and it didn't upload to the repo. But I'll change for consistency.

Reply:
Done